pkgs/os-specific/linux/hyperv-daemons/default.nix

```diff
@@ -31,8 +41,7 @@ let
     '';
 
     postFixup = ''
-      # kvp needs to be able to find the script(s)
-      wrapProgram $out/bin/hv_kvp_daemon --prefix PATH : $out/bin
+      wrapProgram $out/bin/hv_kvp_daemon --prefix PATH : ${lib.makeBinPath [ gawk iproute ]}
```

Comment:
should this list not also include `$out/bin`?

Reply:
We patch the paths earlier, but in order to not fail spectacularly in the future if things change, I've added both bin and libexec. Thanks.